tests/kuttl/common/scripts/rotate_token.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -x
+
+TMP_SECRET_FILE="/tmp/keystone-secret.yaml"
+
+generate_secret_yaml() {
+    cat <<EOF > $TMP_SECRET_FILE
+apiVersion: v1
+kind: Secret
+metadata:
+    name: keystone
+    namespace: keystone-kuttl-tests
+    annotations:
+        keystone.openstack.org/rotatedat: "2009-11-10T23:00:00Z"
+EOF
+}
+
+for rotation in {1..6}; do
```

Comment:
We should do exactly 3 rotations, right? The default is to store 5 keys, and we already do 1 rotation in a previous step. So discounting the staging key, we are left with 3

Reply:
We could try flushing the memcached cache by running
`
oc rsh -n barbican-kuttl-tests memcached-0 bash -c 'echo flush_all > /dev/tcp/localhost/11211'
`
I don't think anything else should be caching the token...

Reply:
The command did not do anything in my env (using correct namespace). I check the files where the keys are saved within a pod, and they are rotated according to the secret, so that looks good.

@stuggi  iirc you helped to implement the fernet key rotation, do you have any idea what can be happening?

Reply:
I'm approving this for now, since these tests is a step forward, but I'm also adding a spike, to figure this out.

Reply:
Yes, thanks for the spike to not let this slip away.